test/fixtures/09-actions/config.ts

```diff
@@ -0,0 +1,51 @@
+import type { ConfigLoader } from '@balena/pinejs';
+import { v1AbstractSqlModel, v1Translations } from './translations/v1/index.js';
+
+const apiRoot = 'actionsUniversity';
+const modelName = 'actionsUniversity';
+const modelFile = import.meta.dirname + '/actionsUniversity.sbvr';
+
+export default {
+	models: [
+		{
+			modelName,
+			modelFile,
+			apiRoot,
+		},
+		{
+			apiRoot: 'v1actionsUniversity',
```

Comment:
I am using the full name here instead of just `v1` just because I wanted the tests to  [be able to infer the PinejsClient typings](https://github.com/balena-io/pinejs/pull/893/files#diff-27e738a0525e0effbc7a062e4fa79c0874932afe5f3be566ee10d6308000eaa7R5-R9) without overiding the type for any model called v1 (which ended up breaking the typings of the other tests so this is just a namespacing)

Reply:
I would say this justifies being a comment in the src b/c otherwise nobody is going to remember the reason by just looking the code.